### PR TITLE
Add connection_limit functionality for PostgreSQL users resolves #1570

### LINKED ISSUE #1570: Add support for CONNECTION LIMIT to postgresql_user
See:
http://www.postgresql.org/docs/9.4/static/sql-createrole.html

for documentation (everything else here is covered by ansible, I think). Should I submit a PR?

### PR DESCRIPTION
**Pull request by schana**
Resolves #1570

### CODE REVIEW

**Comment by robynbergeron** [comment]
Hi @lambeau -- thanks for submitting this PR. 

@lorin -- Please review this PR to make sure it adheres to the following guidelines:

http://docs.ansible.com/developing_modules.html#module-checklist

If it passes these guidelines, and if you believe it’s a good PR otherwise, please add a comment with "shipit" in the text, and we will flag it for inclusion.

If it fails these guidelines, please add a comment with "needs_revision" in the text, along with the needed changes, and we will flag it for revision.

**Comment by gregdek** [comment]
@lorin has advised us that he is no longer maintaining this module, so we've added this back to the list of modules maintained by the core team. It can, therefore, take a while for patches to be reviewed. Thanks for your patience here.

**Comment by gregdek** [comment]
Thanks @schana for this PR. Unfortunately, it is not mergeable in its current state due to merge conflicts. Please rebase your PR. When you are done, please comment with text 'ready_for_review' and we will put this PR back into review.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by gregdek** [comment]
@schana A friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open pending your changes. When you do address the issues, please respond with ready_for_review in your comment, so that we can notify the maintainer.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by schana** [comment]
ready_for_review

**Comment by gregdek** [comment]
Thanks @schana for this PR. This module is maintained by the Ansible core team, so it can take a while for patches to be reviewed. Thanks for your patience.

Core team: please review according to guidelines (http://docs.ansible.com/ansible/developing_modules.html#module-checklist) and comment with 'needs_revision' or merge as appropriate.

[This message brought to you by your friendly Ansibull-bot.]

File: database/postgresql/postgresql_user.py [hunk 1]
```diff
@@ -274,8 +282,9 @@ def user_alter(cursor, module, user, password, role_attr_flags, encrypted, expir
                     role_attr_flags_changing = True
 
         expires_changing = (expires is not None and expires == current_roles_attrs['rol_valid_until'])
+        connection_limit_changing = (connection_limit is not None and connection_limit == current_roles_attrs['rol_conn_limit'])
```
Comment (gregswift) [inline database/postgresql/postgresql_user.py:285]: Role should be singular.

```
current_role_attrs
```

This calls out that line 284 is actually broken in its existing state.

**Review by gregswift — CHANGES_REQUESTED** [review]
Was just wondering if someone had started work on this cause I needed it.  Overall this looks good, aside from one typo that lead to finding an existing bug.

needs_revision

**Pull request closed without merging**